persistence-modules/hibernate-enterprise/src/test/java/com/baeldung/hibernate/HibernateSpatialIntegrationTest.java

```diff
@@ -9,23 +9,23 @@
 import java.net.URL;
 import java.util.Properties;
 
-import javax.persistence.Query;
+import jakarta.persistence.Query;
 
 import org.hibernate.Session;
 import org.hibernate.Transaction;
 import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
+import org.locationtech.jts.geom.Coordinate;
+import org.locationtech.jts.io.ParseException;
```

Comment:
just check if we can use the Java libraries https://docs.oracle.com/javase/8/docs/technotes/guides/2d/spec/j2d-geom.html

Reply:
Sure, I will try.

Reply:
`Coordinate` class doesn't exists in Java Library.